### ISSUE TITLE
save_parameters implementation wastes RAM

### ISSUE THREAD

**Issue by simonepri** [thunlp/OpenKE#159]
The [`save_parameters`](https://github.com/thunlp/OpenKE/blob/86cba3e13b527b94e93178eb347c54cb2a4740a7/config/Config.py#L259) could be rewritten in a way that it does not load the whole json in memory before writing it to a file

**Comment by LiuYuemei111** [comment]
You are right! We will consider your valuable suggestion in future updates
